src/main/java/io/ebeaninternal/server/type/ScalarTypeFile.java

```diff
@@ -97,6 +97,9 @@ public void bind(DataBind b, File value) throws SQLException {
     } else {
       try {
         // stream from our file to the db
+        // This stream doesn't seem to be closed anywhere.
+        // It certainly can't be closed from here, but where then?
```

Comment:
What I mean is: it seems like the way the code works is to pass along an InputStream, set it and use that later whenever necessary. Meaning it has to remain open and cannot be closed.

Reply:
This is really very tricky... I've written a small test and the stream stays open.
As far as I understand, the stream must not be closed until the data is in the database (i.e. until executeUpdate/executeBatch is called)
My current plan is to keep a reference of all open streams in DataBind/BatchedPstmt